mesa/space.py

```diff
@@ -330,14 +333,32 @@ def get_neighbors(
                             Otherwise,
                             return surrounding cells only.
             radius: radius, in cells, of neighborhood to get.
-
         Returns:
             A list of non-None objects in the given neighborhood;
             at most 9 if Moore, 5 if Von-Neumann
             (8 and 4 if not including the center).
         """
         return list(self.iter_neighbors(pos, moore, include_center, radius))
 
+    def inbound_ranges(self, c: int, c_max: int, radius: int) -> tuple[range]:
```

Comment:
Maybe we should make it an internal method, prefixed with `_`, and also say in the docstring that it is currently only used in `get_neighborhood`.